src/App.tsx

```diff
@@ -34,10 +35,11 @@ const App: FC = () => {
   const { pathname } = useLocation();
   const { client } = useDeskproAppClient();
   const { unlink, isLoading: isLoadingUnlink } = useUnlinkContact();
+  const { unlinkCompany, isLoading: isLoadingUnlinkCompany } = useUnlinkCompany()
   const isAdmin = useMemo(() => pathname.includes("/admin/"), [pathname]);
   const isLoading = useMemo(() => {
-    return !client || isLoadingUnlink
-  }, [client, isLoadingUnlink]);
+    return !client || isLoadingUnlink || isLoadingUnlinkCompany
+  }, [client, isLoadingUnlink, isLoadingUnlinkCompany]);
```

Comment:
This doesn't need to be a memo as it's super quick to compute.

Reply:
done